### ISSUE TITLE
object has no attribute 'working_directory' when running cd info

### ISSUE THREAD

**Issue by srussel** [whipper-team/whipper#375]
When running ```whipper cd info``` I get
```
Traceback (most recent call last):
  File "/usr/bin/whipper", line 11, in <module>
    load_entry_point('whipper==0.7.3', 'console_scripts', 'whipper')()
  File "/usr/lib/python2.7/dist-packages/whipper/command/main.py", line 43, in main
    ret = cmd.do()
  File "/usr/lib/python2.7/dist-packages/whipper/command/basecommand.py", line 139, in do
    return self.cmd.do()
  File "/usr/lib/python2.7/dist-packages/whipper/command/basecommand.py", line 139, in do
    return self.cmd.do()
  File "/usr/lib/python2.7/dist-packages/whipper/command/cd.py", line 137, in do
    if self.options.working_directory is not None:
AttributeError: 'Namespace' object has no attribute 'working_directory'
```

The problem appears to be the Info class in cd.py is missing some of the options. I tried copying the arguments from the RIp class into Info.add_arguments and it works. I needed -O, -W, --disc-template, and -o to make it work.

**Comment by srussel** [comment]
I also noticed once I got cd info to run it always ejects the CD. I see the Info class has ```eject = false``` but it still ejects.

**Comment by JoeLametta** [comment]
Fixed with #410.
